cartography/intel/gcp/bigtable.py

```diff
@@ -82,6 +82,9 @@ def get_bigtable_clusters(bigtable: Resource, bigtable_instances: List[Dict], pr
                         cluster['instance_id'] = instance['id']
                         cluster['instance_name'] = instance.get('name')
                         cluster['id'] = cluster['name']
+                        x = cluster.get("location", "global").split("/")
+                        x = x[x.index("locations") + 1].split("-")
+                        cluster['location'] = f"{x[0]}-{x[1]}"
```

Comment:
if there's no location value for cluster and x = global, x[1] would throw error after the split by -. Can you handle this?